addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```
@@ -273,3 +262,10 @@ $badge-padding-x: 0.4em !default;
 
 // Placeholder color 
 $input-placeholder-color: $gray-300 !default;
+
+// Card
+
+$card-spacer-y:                     $spacer !default; // BS Default
```

Some remarks here:

- We don't format variables definition indentation to match a layout
 ```suggestion
$card-spacer-y: $spacer !default; // BS Default
```
- Re-define `$card-bg` it's probably unnecessary, am I right?
- Always add the `!default` directive in bootstrap overridden otherwise these variables cannot be overridden in enterprise or dark-mode

---

if i remove the $card-bg, the background becomes gray because of the overriden in enterprise 
$card-bg: $o-gray-100 !default;

---

mmm... I see that in enterprise`$card-bg` is set to  grey since a very long time.
The [commit description](https://github.com/odoo-dev/enterprise/commit/eb6f248751f56aa9895994b3a1fc245c229bab50#diff-05313bcb5a2a0755d1358c6972297863ecf7cdf778a48f54473b7aa300546647R153) doesn’t say why , the only explanation I can think of is that we needed  cards to contrast over white, and it should not be the case anymore... but I have a very bad memory. 

@qsm-odoo , does it ring a bell to you? 

@morm-odoo , let's wait for qsm, but I think you can  remove `$card-bg` definitions in both community and enterprise, so we"ll inherit BS default (white).

PS. Btw, the method you used is NOT the right one to handle enterprise/community overrides ;)
If you need enterprise to use a specific value you should set it in enterprise directly rather than remove `!default ` from community ;)

---

Ouf, no sorry, it's been a long time since I stop tracking backend design changes :grimacing: In general, I don't think my old knowledge about the way the backend works is worth something anymore :grimacing: